src/api/application/application.ts

```diff
@@ -139,7 +139,7 @@ export class Application extends Base {
      * @return {Promise.Array.<_Window>}
      */
     public getChildWindows(): Promise<Array<_Window>> {
-        return this.wire.sendAction('get-child-windows', this.identity)
+        return this.wire.sendAction<string[]>('get-child-windows', this.identity)
```

Comment:
string[] => _Window[]

Reply:
the SendAction just returns a list of lists of of window names, not a _Window object

Reply:
What, no. 

please test the API and review the return value:
http://cdn.openfin.co/jsdocs/stable/tutorial-application.getChildWindows.html